src/net/sourceforge/kolmafia/textui/Parser.java

```diff
@@ -773,7 +773,7 @@ private Function parseFunction( final Type functionType, final Scope parentScope
 				vararg = true;
 			}
 
-			Variable param = this.parseVariable( paramType, null );
+			Variable param = this.parseVariable( paramType, parentScope, false );
```

Comment:
Does this fix a bug (null -> parentScope)?

(If so, can you add a test case exercising that?)

Reply:
No. This fixes a *future* bug.

When we will keep parsing even after seeing an error, if that parameter is null, we get a NPE.